sdk/python/foundation-models/benchmarking/evaluation_pipelines/azure_openai/boolq.yaml

```diff
@@ -0,0 +1,210 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: boolq_aoai
+description: Azure OpenAI eval on BoolQ
+inputs:
+  endpoint_url: https://resource-name.openai.azure.com/openai/deployments/gpt-4/chat/completions?api-version=2023-07-01-preview
+  ws_connection_name: <connection name>
+  payload_pattern: '{"messages": ###<prompt>, "temperature": 0.0, "max_tokens": 10, "top_p": 1.0, "frequency_penalty": 0.0, "presence_penalty": 0.0, "stop": null}'
+  sample_ratio: 0.01
```

Comment:
Should we set this to 1 in all of the pipelines to ensure anyone running from CLI gets full benchmark by default?

Reply:
Done